awx/main/models/ha.py

```diff
@@ -314,7 +315,8 @@ def save_health_data(self, version=None, cpu=0, memory=0, uuid=None, update_last
                 self.node_state = Instance.States.READY
                 update_fields.append('node_state')
         else:
-            self.mark_offline(perform_save=False, errors=errors)
+            fields_to_update = self.mark_offline(perform_save=False, errors=errors)
+            update_fields.extend(fields_to_update)
```

Comment:
duplicates in `update_fields` aren't a problem, right?

Reply:
yeah I was wondering this to and checked the raw sql with and without duplicate update_fields. They yield the same sql. So I think django is handling duplicates somewhere along the way